roles/openshift_node_group/templates/node-config.yaml.j2

```diff
@@ -56,6 +56,7 @@ masterClientConnectionOverrides:
   contentType: application/vnd.kubernetes.protobuf
   qps: 20
 masterKubeConfig: node.kubeconfig
+nodeName: {{ openshift.node.nodename }}
```

Comment:
No, we should *never* set node name here.  This is wrong.

Reply:
Node name is calculated by the node automatically during bootstrapping.  This role is only used for node bootstrapping.

Reply:
That seems to break Fedora Atomic test. Nodes are `ocp-node1` there, but CSR is created for its IP. Probably the correct way to fix it is to update CSR code then